spec/tarpon/client/subscriber_spec.rb

```diff
@@ -13,6 +13,27 @@
       end
     end
 
+    describe '.attributes' do
+      it_behaves_like 'an http call to RevenueCat', method: :post, api_key: :public, response: :custom do
+        let(:uri) { "#{described_class.base_uri}/subscribers/#{app_user_id}/attributes" }
+        let(:body) do
+          {
+            '$email': {
```

Comment:
It's missing the `attributes` key here.

Reply:
I don't think so, since we're wrapping the data in a Hash with the `attributes` key here: https://github.com/fishbrain/tarpon/blob/54da0d1d2e649426b212ce4ea96e5f82540d8218/lib/tarpon/request/subscriber/attribute.rb#L13-L15

Reply:
Ok we figured it out in [46c4d0a](https://github.com/fishbrain/tarpon/pull/18/commits/46c4d0afcb0b003af63b2a0ea5b05a014f67764f) :)